web/src/components/admin/BrandSettings/BannerEditor/BannerEditor.tsx

```
@@ -0,0 +1,213 @@
+      );
+      return;
+    }
+
+    const accepted = ["image/png", "image/jpeg"].reduce(
+      (prev: string[], curr: string) => {
+        const extensions = mime[curr]?.extensions;
+        if (!extensions) {
+          return prev;
+        }
+
+        return [...prev, ...extensions];
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)